### PR TITLE
create yaml file from jobids in resultpath

### PR DESCRIPTION
**Pull request by hassanfa**
### This PR:

(If this is a release PR, no need to add following. Leave this part empty)
(Use the following lines to create a PR text body. Make sure to remove all non-relevant one after you're done)
(Repeat each field as many times as necessary)

Added: for new features.
Changed: for changes in existing functionality.
Deprecated: for soon-to-be removed features.
Removed: for now removed features.
Fixed: for any bug fixes.
Security: in case of vulnerabilities.

### Review and tests: 
- [ ] Tests pass
- [ ] Code review
- [ ] New code is executed and covered by tests, and test approve

### CODE REVIEW

File: BALSAMIC/commands/run/analysis.py [hunk 1]
```diff
@@ -197,8 +197,7 @@ def analysis(context, snake_file, sample_config, run_mode, cluster_config,
     if run_analysis:
         jobid_file = os.path.join(
             logpath, sample_config["analysis"]["case_id"] + ".sacct")
-        jobid_dump = os.path.join(
-            resultpath, sample_config["analysis"]["case_id"] + "_jobids.yaml")
+        jobid_dump = os.path.join(resultpath, account + "_jobids.yaml")
```
Comment (Mropat) [inline BALSAMIC/commands/run/analysis.py:200]: can it be strictly slurm_job_ids.yaml?

Comment (hassanfa) [reply BALSAMIC/commands/run/analysis.py]: depending on profile used, it will be `slurm_jobids.yaml`, `qsub_jobids.yaml` or `drmaa_jobids.yaml` (in dev)

Comment (hassanfa) [reply BALSAMIC/commands/run/analysis.py]: we use slurm, so it'll be `slurm_jobids.yaml`

**Comment by hassanfa** [comment]
@ashwini06 I am merging this since production needs it for trailblazer integration.

**Comment by hassanfa** [comment]
job ids look like following (partially started workflow):

image

**Review by ashwini06 — COMMENTED** [review]
👍

**Pull request merged**